src/cogent3/align/progressive.py

```diff
@@ -90,6 +90,10 @@ def TreeAlign(
         dists = dcalc.get_pairwise_distances().to_dict()
         tree = NJ.nj(dists)
 
+    for edge in tree.postorder():
+        if edge.name != "root":
```

Comment:
You may need converting to lower case before comparing strings.

Reply:
Not for this case, the "root" name is created by the instance, not the user.

Reply:
OK. Thank you.